apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.cs

```diff
@@ -87,7 +87,22 @@ public abstract partial class BigQueryClient : IDisposable
         /// no credentials are specified.
         /// </summary>
         /// <remarks>
-        /// The credentials are scoped as necessary.
+        /// <para>
+        /// If the supplied credentials support scoping, the following scopes are specified, overwriting any
+        /// previously-specified scopes.
+        /// </para>
+        /// <list type="bullet">
+        /// <item><description>https://www.googleapis.com/auth/bigquery"</description></item>
+        /// <item><description>https://www.googleapis.com/auth/bigquery.insertdata"</description></item>
+        /// <item><description>https://www.googleapis.com/auth/devstorage.full_control"</description></item>
+        /// <item><description>https://www.googleapis.com/auth/cloud-platform"</description></item>
+        /// </list>
+        /// <para>
+        /// If you require a different set of scopes, please use create and scope
```

Comment:
nit: please use create -> please create

Reply:
Whoops, done - pushed, and will merge on green.